vega_sim/service.py

```diff
@@ -1012,6 +1012,44 @@ def update_network_parameter(
         self.wait_fn(60)
         self.wait_for_thread_catchup()
 
+    def update_market_state(
+        self,
+        market_id: str,
+        proposal_key: str,
+        market_state: vega_protos.governance.MarketStateUpdateType,
+        price: Optional[str] = None,
+        wallet_name: Optional[str] = None,
+        approve_proposal: bool = True,
+    ):
```

Comment:
I'm not too fussed but I thought we were having optional datetimes for proposal methods now?